src/utils/helpers.js

```diff
@@ -0,0 +1,66 @@
+const { convertToCase } = require('../convertToCase/convertToCase');
+const { supportedCases, statusCodes } = require('./constants');
+
+function parseUrl(req) {
+  const splitted = req.url.split('?');
+  const originalText = splitted[0].slice(1);
+  const targetCase = new URLSearchParams(splitted[1]).get('toCase');
+
+  return { targetCase, originalText };
+}
+
+function sendResponse(res, statusCode, data) {
+  res.setHeader('Content-Type', 'application/json');
+  res.statusCode = statusCode;
+  res.end(JSON.stringify(data));
+}
+
+function sendTransformedResponse(res, targetCase, originalText) {
+  const { originalCase, convertedText } = convertToCase(
+    originalText,
+    targetCase,
+  );
+
+  sendResponse(res, statusCodes.OK, {
+    originalCase,
+    targetCase,
+    originalText,
+    convertedText,
+  });
+}
+
+function validate(targetCase, originalText) {
+  const errors = [];
+
+  if (!targetCase) {
+    errors.push({
+      message: `"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+    });
+  }
+
+  if (!originalText.trim()) {
+    errors.push({
+      message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+    });
+  }
+
+  if (
+    targetCase &&
+    !supportedCases.some(
+      (supportedCase) => supportedCase === targetCase.toUpperCase(),
+    )
+  ) {
+    errors.push({
+      message: `This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.`,
+    });
```

Comment:
Also would be great to create an object with all error messages as constant

Reply:
Fixed it, thanks!